Multisig/UI/UI Library/Safe Settings/AdvancedSafeSettingsView.swift

```diff
@@ -42,12 +38,14 @@ struct AdvancedSafeSettingsView: View {
     }
     
     var fallbackHandlerView : some View {
-        Group {
-            if (safe.fallbackHandler ?? "").isEmpty  {
-                 EmptyView()
+        let fallbackHandler = safe.fallbackHandler ?? ""
```

Comment:
I would add the `isDefaultFallbackHandler` to the Safe model, and then just use that property in the condition below.

```
let title = safe.isDefaultFallbackHandler ? "..." : "Unknown"
```

Reply:
I don't think so, this feature is related to address not safe, maybe we need to check if given address is default fallback handler or not

Reply:
per discussion: this is related to the Safe contract (default values) and similar to the decision about the supported master copies. So, creating a separate SafeContract class would be the way to go.